src/freezing/freezing_main.py

```diff
@@ -35,6 +35,16 @@ def run_freezing(
         constructed_df (pd.DataFrame): As main_snapshot but with records amended
             and added from the freezing files.
     """
+    # return frozen snapshot if config allows
+    run_first_snapshot_of_results = config["global"]["run_first_snapshot_of_results"]
```

Comment:
As discussed, this should be run_frozen_data = True, and read in a frozen csv (skipping staging). run_first_snapshot_of_results = True, is running from the json snapshot (not skipping staging)).